doc/services/tracing/index.rst

```diff
@@ -191,14 +191,14 @@ Using Tracing
 The sample :zephyr_file:`samples/subsys/tracing` demonstrates tracing with
 different formats and backends.
 
-To get started, the simplest way is to use the CTF format with the ``native_posix``
+To get started, the simplest way is to use the CTF format with the ``native_sim``
```

Comment:
I think it also needs updating in the README of the actual sample ("Usage for POSIX Tracing Backend" section, to make sure this really becomes the official recommendation for "the simplest way" of doing things).